q2doc/usage/utils.py

```diff
@@ -5,3 +5,9 @@ def get_docname(node):
     root, doc_name = [Path(p) for p in Path(node.source).parts[-2:]]
     doc_name = Path(doc_name.stem)
     return root, doc_name
+
+
+def get_ext(source):
+    if source == "metadata" or source == "init_metadata":
```

Comment:
I'm not seeing `"metadata"` as a valid record source anywhere in the based usage drivers - are you observing this source somewhere in your tests? If so, where?

Reply:
This function is handling strings derived from either a `ScopeRecord` or `FactoryNode.factory` (see [here](https://github.com/andrewsanchez/sphinx-ext-qiime2/blob/usage-ext/q2doc/usage/usage.py#L78)) which can either be `"data"` or `"metadata"`, e.g.:

```
.. usage::
   :factory: metadata
   :name: md1

   def md1_factory():
       return Metadata(pd.DataFrame({'a': ['1', '2', '3']},
                                   index=pd.Index(['0', '1', '2'],
                                                   name='id')))
```